doc/example/platform.txt

```diff
@@ -0,0 +1,41 @@
+
+Platform specification
+============================
+
+Basic multi-platform example
+----------------------------
+
+Assuming the following layout::
+
+    tox.ini      # see below for content
+    setup.py     # a classic distutils/setuptools setup.py file
+
+and the following ``tox.ini`` content::
```

Comment:
You can make this syntax highlighted by writing it like this:

```
and the following ``tox.ini`` content:

.. code-block:: ini

    [tox]
    # platform specification support is available since version 2.0
```

(mind the single `:` in `content:`)